op_bake.py

```
@@ -543,11 +543,28 @@ def assign_tune_materials(obj, setup_bake_nodes=False):
             for i in range(len(bset.objects_low)):
-                obj_low = bset.objects_low[i]
+                obj_low = None
+
                 try:
-                    if obj_low.name not in bpy.data.objects: raise ReferenceError
-                except ReferenceError:
-                    continue
+                    candidate = bset.objects_low[i]
+                    if candidate.name in bpy.data.objects:
+                        obj_low = candidate
+                except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)